docs/make_work.jl

```diff
@@ -166,12 +166,30 @@ function doit(
         end
         src = normpath(root, file)
         dst = normpath(dstbase, relpath(root, srcbase), file)
-        cp(src, dst; force=true)
+        if endswith(file, ".md")
```

Comment:
Not an objection, just wondering: why not just symlink all the files?

Reply:
I had it first with only symlinks, but that leaded to a red wall on https://github.com/oscar-system/Oscar.jl/actions/workflows/pages/pages-build-deployment, so it basically blocked all updates to the docs from being published.
How I try to explain it: In the building stage (converting md to HTML), Documenter is fine with files not in the project folder it is called on. However, in later stages (where e.g. `*.js` are still there), it does not like symlinks outside of the project folder anymore. And thinking again about it, that would be kind of hard to check-in into git anyway :joy: